packages/esm-ward-app/src/ward-workspace/ward-patient-notes/notes-action-button.extension.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ActionMenuButton, launchWorkspace, StickyNoteAddIcon } from '@openmrs/esm-framework';
+
+export default function WardPatientNotesActionButton({ additionalProps: workspaceProps }: { additionalProps: object }) {
```

Comment:
We are not getting any props in the Action Button.
Please remove this as this will pass unnecessary props inside the `launchWorkspace` function.
Thanks!